router/pump.go

```diff
@@ -89,6 +89,15 @@ func (p *LogsPump) Setup() error {
 	return nil
 }
 
+func (p *LogsPump) rename(event *docker.APIEvents) {
+    p.mu.Lock()
+    defer p.mu.Unlock()
+    container, err := p.client.InspectContainer(event.ID)
```

Comment:
Logspout wouldn't require docker `>= 1.10.0` since go-dockerclient maintains forward and backward compatibility.